scripts/cloud-core.sh

```diff
@@ -0,0 +1,170 @@
+#! /bin/bash
+
+set -e 
+
+# Enter the IP address of the a: Cloud Core Node , b: Edge Node with Control Plane , c: Edge Worker node 
+declare -x a=
+declare -x b=
+declare -x c=
+
+#Enter kubernetes version
+
+v='1.21.1-00'
+
+#To kill running process of cloudcore
+cloudcore=`ps -aef | grep _output/local/bin/cloudcore | grep -v sh| grep -v grep| awk '{print $2}'`
+
+
+    pushd $HOME
+#---------------------------------------------------------------------
+echo '*****SETTING UP THE HOSTNAME NODE-A*****'
+    sudo hostnamectl set-hostname node-a
+#---------------------------------------------------------------------
+echo '*****DISABLING FIREWALL*****'
+    sudo ufw disable
+    sudo swapoff -a
+#---------------------------------------------------------------------
+ip_tables(){
+    echo '*****LETTING IPTABLES SEE BRIDGED TRAFFIC*****'
+    sudo modprobe br_netfilter
+    sudo apt-get -y update
+    echo -e 'br_netfilter' | cat > /etc/modules-load.d/k8s.conf
+    echo -e 'net.bridge.bridge-nf-call-ip6tables = 1\nnet.bridge.bridge-nf-call-iptables = 1' | cat >> /etc/sysctl.d/k8s.conf
+    sysctl --system
+}
+#----------------------------------------------------------------------
+docker_install(){
+    echo '*****INSTALLING DOCKER*****'
+    sudo apt-get update -y
+    if [ "$(which docker)" != "" ]
+    then
+       echo "Docker is already installed" 
+    else
+       sudo apt-get install docker.io -y 
+    fi
+	if [ "$(which vim)" != "" ]
+    then
+        echo "VIM is already installed" 
+    else
+        sudo apt-get install vim -y 
+    fi
+}	
+#----------------------------------------------------------------------------
+kube_packages(){
+    echo '*****INSTALLING KUBEADM, KUBELET AND KUBECTL*****'
+    sudo apt-get update -y
+    if  [ "$(which apt-transport-https ca-certificates curl)" != "" ]
+    then
+        echo "apt-transport-https ca-certificates curl is already installed"  
+    else
+        sudo apt-get install apt-transport-https ca-certificates curl -y  
+    fi
+    sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+    echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+    sudo apt-get update
+    sudo apt-get install -y kubelet=$v kubectl=$v kubeadm=$v
+    sudo apt-mark hold kubelet kubeadm kubectl
+    systemctl start docker.service
+    systemctl enable docker.service
+}
+#----------------------------------------------------------------------------
+kube_cluster(){
+    echo '*****STARTING CLUSTER USING KUBEADM*****'
+    echo y | kubeadm reset     # To remove the existing cluster if running
+    sudo rm -rf $HOME/.kube/config
+    kubeadm init
+    sleep 5s
+    export KUBECONFIG=/etc/kubernetes/admin.conf
+    sleep 5s
+    systemctl restart kubelet
+    sleep 20s
```

Comment:
is this sleep needed? once kubelet restart is back, it'll be up running. what's the sleep for?